searchbase/README.md

```diff
@@ -248,3 +248,214 @@ class DefaultUriPolicy implements UriPolicy {
   }
 }
 ```
+
+### Facet example extended to use SearchStateController to handle state changes side-effects
+
+This example is a version of the previous example which uses `SearchStateController` class to watch for state changes and perform relevant side-effects.
```

Comment:
> previous example

Can you link the example?

Reply:
Fixed here 👉  https://github.com/appbaseio/flutter-searchbox/pull/21/commits/1bc12704ca626f05c9bceb0804e5dff385bae78c